src/bidiMapper/domains/context/browsingContextProcessor.ts

```diff
@@ -38,15 +38,21 @@ export class BrowsingContextProcessor {
     );
   }
 
+  async #removeChildrenContexts(context: BrowsingContextImpl): Promise<void> {
+    // Remove context's children.
+    for (let child of context.children) {
+      await this.#removeContext(child.contextId);
```

Comment:
Is the order of remove important? Perhaps, `await Promise.all(context.children.map(child => this.#removeContext(child.contextId)))`?

Reply:
done